### DISCUSSION TITLE
Can the docX be used to fill values in a PDF ?

### DISCUSSION THREAD

**Discussion by aviral282** [dolanmiu/docx#1269] [Q&A] [answered]
Hello Everyone,

I am testing my integration with docX the library is amazing in building the documents.

However, I was wondering if it is possible to populate a pdf( A pdf form with empty fields to be populated) using docX, or is there any other resource available to achieve this.

Thank you
[sample.pdf](https://github.com/dolanmiu/docx/files/7435235/sample.pdf)

**Comment by dolanmiu** [comment, marked as answer]
Not possible im afraid

This library can only be used to generate docx files

**Comment by aviral282** [comment]
That's perfect, Thanks I'll try to look for alternatives to achieve this.
Thank you
